Apps/VisionOS/README.md

```diff
@@ -0,0 +1,59 @@
+# VisionOS Sample App
+
+The VisionOS Sample App is a practical demonstration of how to integrate and utilize the Customer.io Data Pipelines iOS/Swift SDK.
+
+## Who Is This App For?
+
+This sample app is designed for VisionOS and iOS developers who are looking to get hands-on experience with Customer.io's code samples. It serves as a tool to validate, debug, or enhance their understanding of the different Customer.io APIs.
+
+## Run the App
+
+Before running the app, ensure you have one of the following prerequisites:
+* Apple Vision Pro simulator, which can be downloaded from Xcode.
+* An Apple Vision Pro device with developer mode enabled.
+
+To run the app, follow these steps:
+
+1. Clone the [customerio-ios](https://github.com/customerio/customerio-ios) repository by executing:
+   ```
+   git clone https://github.com/customerio/customerio-ios
+   ```
+2. Navigate to `<REPO_ROOT_DIR>/Apps/VisionOs` and open the `VisionOs.xcodeproj` file with Xcode.
```

Comment:
Suggestion - Folder name is [VisionOS](https://github.com/customerio/customerio-ios/tree/main/Apps/VisionOS).

```suggestion
2. Navigate to `<REPO_ROOT_DIR>/Apps/VisionOS` and open the `VisionOS.xcodeproj` file with Xcode.
```